File: Blind 75 LeetCode/Pankajavalli RV/Product of Array Except Self/code.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        
        prefix = [1]
        postfix = [1]

        for i in range(0,len(nums) ):
            val = prefix[-1] * nums[i]
            prefix.append(val)
        
        for i in range(len(nums) - 1, -1, -1):                        
            val = nums[i] * postfix[0]
            postfix = [val] + postfix
          
        res = []  
      
        for i in range(0,len(nums)):
            res.append(prefix[i] * postfix[i+1])
        return res

File: Blind 75 LeetCode/Shubham Sharma/Missing number/code.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n=nums.size();
        sort(begin(nums), end(nums));
        for(int i=0; i<n; i++){
           if(nums[i]==i) continue;
           else return i;
        }
        return n;
    }
};